opacity attribute and example added to maping elements - layer, (#100)
Maps4HTML/web-map-doc: docs/elements/layer.md

---

### `opacity`

> The `opacity` attribute is used to set the opacity of the `<layer->` element. Just
> like the opacity slider on the map, adding this attribute can change the initial
> opacity of any layer.

Reviewer: ```suggestion
The `opacity` attribute is used to set the initial opacity of the `<layer->` element.
When the `opacity` attribute is not present, the opacity is set to "1.0" by default.
```

Reviewer: Updated this suggestion to include valid opacity range and mention the layer control opacity input slider control

---

## Examples

### Layer Opacity

> The following example shows `opacity` attribute in a `<layer->`. 

Reviewer: ```suggestion
The following example sets the initial `opacity` for a `<layer->`. 
```

```html
<mapml-viewer projection="CBMTILE" zoom="2" lat="45" lon="-90" controls>
      <layer- opacity = "0.5" label="CBMT" src="https://geogratis.gc.ca/mapml/en/cbmtile/cbmt/" checked></layer->
</mapml-viewer>
```

> 

Reviewer: ```suggestion
---

```
